Converters/Civil3d/Speckle.Converters.Civil3d2024/Speckle.Converters.Civil3d2024.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net48</TargetFramework>
     <PlatformTarget>x64</PlatformTarget>
+    <DefineConstants>$(DefineConstants);CIVIL3D;CIVIL3D2024;CIVIL3D2024_OR_GREATER</DefineConstants>
```

Comment:
Guessing we should have `CIVIL3D2023_OR_GREATER` here too?
Tho, not sure the status of our CIVIL versions... if we don't have other versions yet, then we should just remember to correct the `DefineConstants` later